Modules/SwiftUISupport/StatefulSwiftUIView.swift

```diff
@@ -0,0 +1,28 @@
+import JamitFoundation
+import UIKit
+import SwiftUI
+
+@available(iOS 13.0, *)
+/// Takes a stateful view as a generic type and wraps it into a `UIViewRepresentable`.
+/// The stateful views model can be changed by changing the `model` property.
+public struct StatefulViewRepresentable<ContentView: StatefulViewProtocol>: UIViewRepresentable {
```

Comment:
`StatefulViewRepresentable` sounds a bit like a `protocol`, not like a `struct`. How about `StatefulViewRepresenter`?

I don't have really strong opinions on this, but maybe this is something worth discussing again (maybe with @mrylmz)  before merging this PR and thereby somehow finalizing the name.